diagnostic.md

```diff
@@ -48,11 +52,14 @@ Record your responses inside the fenced code blocks below each question.
     value `'123'` inside a Route?
 
     ```md
-    <!-- your response here -->
+    Not sure what is meant by referencing the value. If you mean how do we get
+    to the data about `/movies/123` other than manually typing it into the url
+    window then I think you just an action inside the model that says to show
+    the component that contains movie data.
```

Comment:
This means within the  Route object. You would access it using `params.movie_id`.